business_rules/actions.py

```diff
@@ -45,6 +51,10 @@ def _validate_action_parameters(func, params):
 def rule_action(label=None, params=None):
     """
     Decorator to make a function into a rule action.
+    If value of params dictionary is field_type, defaultValue is None.
+    If action parameter requires a default value, ActionParam is used as value in parameters' dict.
+    It has field_type and default_value attributes which are used as fieldType and defaultValue values accordingly.
```

Comment:
Could you try to rephrase this? I think it's not clear the explanation.
What do you think about the following?
```
`params` parameter could be one of the following:
1. Dictionary with params names as keys and types as values
Example:
params={
    'param_name': fields.FIELD_NUMERIC,
}

2. If a param has a default value, ActionParam can be used. Example:
params={
    'voucher_template_id': ActionParam(field_type=fields.FIELD_NUMERIC, default_value=123)
}
```